core/Updates/4.0.0-b1.php

```diff
@@ -110,6 +113,34 @@ public function getMigrations(Updater $updater)
         // remove old options
         $migrations[] = $this->migration->db->sql('DELETE FROM `' . Common::prefixTable('option') . '` WHERE option_name IN ("geoip.updater_period", "geoip.loc_db_url", "geoip.isp_db_url", "geoip.org_db_url")');
 
+        // replace days to ... dimensions w/ seconds dimensions
+        foreach (['log_visit', 'log_conversion'] as $table) {
+            $migrations[] = $this->migration->db->addColumn($table, 'visitor_seconds_since_first', VisitorSecondsSinceFirst::COLUMN_TYPE);
+            $migrations[] = $this->migration->db->addColumn($table, 'visitor_seconds_since_order', VisitorSecondsSinceFirst::COLUMN_TYPE);
+        }
+
+        $migrations[] = $this->migration->db->addColumn('log_visit', 'visitor_seconds_since_last', VisitorSecondsSinceLast::COLUMN_TYPE);
```

Comment:
todo: we may need to test if Matomo will automatically merge this add column with above tables or execute it individually. Otherwise need to move it up into the for loop and potentially use `addColumns()`

Reply:
will do use `addColumns()` just to be safe